### PR TITLE
new cask: zekr 1.1.0

### PR DESCRIPTION
**Pull request by yurikoles**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/zekr.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'zekr' do
+  version '1.1.0'
+  sha256 '95c1fc030f250e29ab7227170d8a670d269810f2281ccd3def6ba81604672518'
+
+  # downloads.sourceforge.net/zekr was verified as official when first introduced to the cask
+  url "https://downloads.sourceforge.net/zekr/zekr-#{version}-mac_64.tgz"
+  name 'Zekr'
+  homepage 'http://zekr.org/'
+
+  depends_on cask: 'java'
```
Comment (vitorgalvao) [inline Casks/zekr.rb]: The correct approach is [`depends_on_java` `caveat`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/caveats.md#caveats-mini-dsl).

File: Casks/zekr.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'zekr' do
+  version '1.1.0'
+  sha256 '95c1fc030f250e29ab7227170d8a670d269810f2281ccd3def6ba81604672518'
+
+  # downloads.sourceforge.net/zekr was verified as official when first introduced to the cask
+  url "https://downloads.sourceforge.net/zekr/zekr-#{version}-mac_64.tgz"
```
Comment (vitorgalvao) [inline Casks/zekr.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/zekr.rb [hunk 3]
```diff
@@ -4,10 +4,12 @@
 
   # downloads.sourceforge.net/zekr was verified as official when first introduced to the cask
   url "https://downloads.sourceforge.net/zekr/zekr-#{version}-mac_64.tgz"
+  appcast 'https://sourceforge.net/projects/zekr/rss?path=/Zekr',
+          checkpoint: 'd4156f90a39a16f9247164fa8a5536e6c5e396e55fff047cb7fe86aa2f114643'
   name 'Zekr'
   homepage 'http://zekr.org/'
 
-  depends_on cask: 'java'
+  depends_on_java(6)
```
Comment (vitorgalvao) [inline Casks/zekr.rb]: It needs to be a `caveat`, not a stanza. It’s not supposed to be an integer, but a string. And does this really depend on version 6, or 6 and up?

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/zekr.rb (hunk 3 above)
Comment (yurikoles) [inline Casks/zekr.rb]: how to specify 6+?

Comment (yurikoles) [inline Casks/zekr.rb]: done

**Pull request merged**